core/meta/test/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 
 ROOT_ADD_GTEST(testStatusBitsChecker testStatusBitsChecker.cxx LIBRARIES Core)
 ROOT_ADD_GTEST(testHashRecursiveRemove testHashRecursiveRemove.cxx LIBRARIES Core)
-if(NOT MSVC OR win_broken_tests OR CMAKE_CXX_STANDARD GREATER 14)
+if(NOT MSVC OR win_broken_tests)
```

Comment:
I believe this means the test should be enabled for C++17, even on Windows? @bellenot

Reply:
correct